vim25/soap/client.go

```diff
@@ -86,7 +86,7 @@ type Client struct {
 	Types     types.Func `json:"types"`
 	UserAgent string     `json:"userAgent"`
 
-	cookie          string
+	Cookie          func() any
```

Comment:
Should we document this and indicate that someone can set it to `nil` to avoid participating in the cookie?